ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__arm64-nightly-4.11-upgrade-from-stable-4.10.yaml

```diff
@@ -144,6 +144,7 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      FIPS_ENABLED: "true"
```

Comment:
Unexpected change ?

Reply:
look good to me except this one:) Thank you.

Reply:
Good catch, fixed.